docstrings_to_openapi/to_openapi.py

```diff
@@ -0,0 +1,55 @@
+import json
+import sys
+
+import yaml
+
+from .cli import OPTS
+
+
+def main():
+    """The main function to bulding the OpenAPI doc for the app's API.
+
+    Here starts by mapping the supporting frameworks and docstring styles that
+    can parse controller/view docs.
+
+    Then with the incoming parameters when executing this tool, will call into
```

Comment:
Maybe something like this?

“Based on the parameters the tool was called with, it will then call into the chosen modules to build the openapi documentation.”

Reply:
I combined your 2 suggestions, i think it makes more sense now